### PR TITLE
Finish items Part 2: sharing and NOTIFY support 

### PR DESCRIPTION
**Pull request by daviddahl**
This PR will provide the sharing and NOTIFY support needed to tell users of updated Items that were shared with them

### CODE REVIEW

**Comment by daviddahl** [comment]
I am currently having a great time working out the trigger, procedure and query to notify all the "notifees" when an item is updated. I was having trouble with syntax and getting the returned table rows to be usable inside the trigger. Now it seems to run silently as if the LOOP never engages...
https://github.com/SpiderOak/crypton/pull/351/files#diff-a65deab604601f58f9ca47fa5e31b214R684

**Comment by daviddahl** [comment]
This PR is complete, methinks. create / update / remove Items, share / unshare items. Going to play around with it before review & merging. @devgeeks - If you have time to look at this I'd be much obliged!

File: client/src/session.js [hunk 1]
```diff
@@ -30,6 +32,7 @@
  * @param {Number} id
  */
 var Session = crypton.Session = function (id) {
+  ERRS = crypton.errors;
```
Comment (ecto) [inline client/src/session.js]: No need to do this in the `Session` constructor

File: server/lib/stores/postgres/transaction.js [hunk 2]
```diff
@@ -49,6 +49,9 @@ var transactionQuery = fs.readFileSync(__dirname + '/sql/transaction.sql').toStr
   client.query('listen "container_update"');
 
   client.on('notification', function (data) {
+    if (data.channel != 'container_update') {
```
Comment (ecto) [inline server/lib/stores/postgres/transaction.js]: 'container_update' should be a constant

**Comment by ecto** [comment]
Non-error logging shouldn't go to production for a framework, the developer should decide when to log

**Comment by ecto** [comment]
Nice tests :~)

File: client/src/session.js (hunk 1 above)
Comment (daviddahl) [inline client/src/session.js]: For whatever reason, ERRS was not reliably being set otherwise.

**Comment by daviddahl** [comment]
Yes, there are a few "logs" to excise, I just saw "9 test passed" and stopped to tinker with this in a real app.

File: client/src/session.js (hunk 1 above)
Comment (ecto) [inline client/src/session.js]: Ah, I would be that errors.js is being concatenated after session.js then and isn't loaded when the IIFE is

**Comment by daviddahl** [comment]
I'm merging this as I have been testing it heavily for the past 3 weeks.

**Pull request merged**